src/packages/@library/forms/multi-select/MultiSelect.svelte

```diff
@@ -0,0 +1,285 @@
+<script lang="ts">
+  import dropdown from "$lib/assets/dropdown.svg";
+  import { onDestroy, onMount } from "svelte";
+  import { slide } from "svelte/transition";
+  import closeIcon from "$lib/assets/close.svg";
+
+  interface Data {
+    name: string;
+    id: string;
+    checked: boolean;
+  }
+  /**
+   * Indicates if there is an error.
+   */
+  export let isError: boolean = false;
+  /**
+   * Callback function to handle click events on a tab.
+   */
+  export let onclick: (list: Data[]) => void;
+  /**
+   * The ID of the multi select.
+   */
+  export let id: string;
+  /**
+   * The data for the multi select.
+   */
+  export let data: Array<{
+    name: string;
+    id: string;
+    checked: boolean;
+  }>;
+
+  let isOpen: boolean = false;
+  let list: Data[];
+  if (data && data.length > 0) {
+    list = data;
+  } else {
+    list = [];
+  }
+
+  /**
+   * Toggles the multi select open state.
+   */
+  const toggleDropdown = () => {
+    isOpen = !isOpen;
+  };
+
+  /**
+   * Handles clicks outside the dropdown to close it.
+   * @param  event - The mouse event.
+   */
+  function handleDropdownClick(event: MouseEvent) {
```

Comment:
make it a arrow function, which we are also following everywhere

Reply:
resolved